resolwe_bio/processes/workflows/bbduk_salmon_qc.py

```diff
@@ -0,0 +1,400 @@
+            },
+        }
+
+        if inputs.quantification.consensus_slack:
+            input_salmon["options"][
+                "consensus_slack"
+            ] = inputs.quantification.consensus_slack
+
+        if inputs.reads.type.startswith("data:reads:fastq:paired:"):
+            input_salmon["options"]["gc_bias"] = True
+
+        quantification = Data.create(
+            process=BioProcess.get_latest(slug="salmon-quant"),
+            input=input_salmon,
+        )
+
+        input_seqtk = {
+            "reads": preprocessing,
+            "n_reads": inputs.downsampling.n_reads,
+            "advanced": {
+                "seed": inputs.downsampling.seed,
+                "fraction": inputs.downsampling.fraction,
+                "two_pass": inputs.downsampling.two_pass,
+            },
+        }
+
+        if inputs.reads.type.startswith("data:reads:fastq:single:"):
+            seqtk_slug = "seqtk-sample-single"
+        elif inputs.reads.type.startswith("data:reads:fastq:paired:"):
+            seqtk_slug = "seqtk-sample-paired"
+        else:
+            self.error("Wrong input type.")
+
+        downsampling = Data.create(
+            process=BioProcess.get_latest(slug=seqtk_slug), input=input_seqtk
+        )
+
+        alignment_qc = Data.create(
+            process=BioProcess.get_latest(slug="alignment-star"),
+            input={
+                "reads": downsampling,
+                "genome": inputs.genome,
+            },
+        )
+
+        alignment_qc_rrna = Data.create(
+            process=BioProcess.get_latest(slug="alignment-star"),
+            input={
+                "reads": downsampling,
+                "genome": inputs.rrna_reference,
+            },
+        )
+
+        alignment_qc_globin = Data.create(
+            process=BioProcess.get_latest(slug="alignment-star"),
+            input={
+                "reads": downsampling,
+                "genome": inputs.globin_reference,
+            },
+        )
+
+        qorts = Data.create(
+            process=BioProcess.get_latest(slug="qorts-qc"),
+            input={
+                "alignment": alignment_qc,
+                "annotation": inputs.annotation,
+                "options": {
+                    "stranded": "auto",
+                    "cdna_index": inputs.salmon_index,
+                    "n_reads": 5000000,
+                },
+            },
+        )
+
+        idxstats = Data.create(
+            process=BioProcess.get_latest(slug="samtools-idxstats"),
+            input={
+                "alignment": alignment_qc,
+            },
+        )
+
+        multiqc_inputs = [
+            inputs.reads,
+            preprocessing,
+            quantification,
+            downsampling,
+            alignment_qc,
+            alignment_qc_rrna,
+            alignment_qc_globin,
+            qorts,
+            idxstats,
+        ]
+
+        inputs_multiqc = {"data": multiqc_inputs}
```

Comment:
Use `input_multiqc` to be consistent with the above input dicts. You can probably incorporate the above chunk (`multiqc_inputs`) directly into this dict without an intermediate list.